internal/services/mysql/mysql_flexible_server_configuration_resource_test.go

```diff
@@ -85,6 +85,28 @@ func TestAccMySQLFlexibleServerConfiguration_logSlowAdminStatements(t *testing.T
 	})
 }
 
+func TestAccMySQLFlexibleServerConfiguration_update(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_mysql_flexible_server_configuration", "test")
+	r := MySQLFlexibleServerConfigurationResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.template(data, "slow_query_log", "OFF"),
+			Check: acceptance.ComposeTestCheckFunc(
+				data.CheckWithClient(r.checkValue("OFF")),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.template(data, "slow_query_log", "ON"),
```

Comment:
Can we follow the pattern here in the other tests rather than specifying the template directly?  e.g. `Config: r.characterSetServer(data),`

Reply:
Sure. Updated.

![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/19754191/312499b1-17c9-4bfa-aa2f-aa9c46cd4725)